library/billrep.inc

```diff
@@ -1,165 +1,238 @@
 <?php
-require_once(dirname(__FILE__) . "/../interface/reports/report.inc.php");
+require_once dirname(__FILE__) . "/../interface/reports/report.inc.php";
```

Comment:
This script is gonna take me some time to review/digest
Ongoing review placeholder

Reply:
hi @KyleWiering ,
Still analyzing this script (need a good night of sleep for this; not because of your code, but because of the initial code...). If you want to work on something else while I review this script, just let me know and I can direct you to another project, like this one:
https://github.com/openemr/openemr/issues/2119